integration-testing/green/project.clj

```diff
@@ -0,0 +1,7 @@
+(defproject green "n/a"
+  :description "Please keep me updated - `lein antq` should pass for this project"
+  :managed-dependencies [[com.stuartsierra/dependency "1.0.0"]]
+  :dependencies [[org.clojure/clojure "1.10.3"]
+                 [com.stuartsierra/dependency]]
+  :plugins [[lein-pprint "1.3.2"]
+            [com.github.liquidz/antq "RELEASE"]])
```

Comment:
I want to move this file to `test/resources/integration/green/project.clj` because I'd like to keep all the resources for testing here.
Other files (red1, red2, red3) same here.